alpha/lib/model/DeliveryProfileLiveAppleHttp.php

```diff
@@ -324,5 +256,26 @@ private function buildFlavorBitrateInfoArray($primaryServerStreams)
 		}
 		return $flavorBitrateInfo;
 	}
+	
+	private function isAudioStream($primaryServerStreams, $backupServerStreams)
+	{
+		if(count($primaryServerStreams))
+		{
+			/* @var $kLiveStreamParams kLiveStreamParams */
+			$kLiveStreamParams = reset($primaryServerStreams);
+			if(!$kLiveStreamParams->getHeight() && !$kLiveStreamParams->getWidth())
```

Comment:
In case of audio stream the width and height will a;ways be undefined so no case to check for cases where value is 0.

The use case of streaming different streams to primary and backup is not a normal situation and is not supported